.github/workflows/grid_client_nightly.yml

```diff
@@ -145,6 +151,7 @@ jobs:
             > **Details on failed run**: https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}
 
             - **Dynamic Single Vm**: ${{ steps.single_vm.outcome }}
+            - **ZOS3 Lite Mycelium**: ${{ steps.zos3lite.outcome }}
             - **Multiple Vm**: ${{ steps.multiple_vm.outcome }}
             - **Kubernetes**: ${{ steps.k8s.outcome }}
```

Comment:
A new test for mutiple VMs zos3 should be added here also

Reply:
done in the new pr